packages/dropdown/src/Default/Default.types.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface DropDownItemProps {
+  id: string;
+  label: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any | undefined;
+  length: number;
+}
+
+export interface DropDownContainerProps {
```

Comment:
Removing the dropdown object and keeping only the `items` you should be good to remove this interface.

Also, the `items` in the `DropdownDefaultCommons` should be an array of `DropDownItemProps`:

```ts
export interface DropdownDefaultCommons {
  // appearance?: 'Primary' | 'Outline';
  theme?: 'Default' | 'Dark';
  size?: 'Sm' | 'Md' | 'Lg';
  prefix?: string;
  children: React.ReactNode;
  items: DropDownItemProps[];
}
```